add definition WS2812_BYTE_ORDER to fix RGB LED issues (#10184)
qmk/qmk_firmware: docs/ws2812_driver.md

#define WS2812_TRST_US 80
```

> ### Byte Order

Reviewer: This should sit under `Driver configuration -> All drivers`
```suggestion
#### Byte Order
```

Some variants of the WS2812 may have their color components in a different physical or logical order. For example, the WS2812B-2020 has physically swapped red and green LEDs, which causes the wrong color to be displayed, because the default order of the bytes sent over the wire is defined as GRB.
In this case, you can change the byte order with the following define:

```c
#define WS2812_BYTE_ORDER WS2812_BYTE_ORDER_RGB
```

> 

Reviewer 2: ```suggestion

| Byte order                        | Known devices                 |
|-----------------------------------|-------------------------------|
| `WS2812_BYTE_ORDER_GRB` (default) | Most WS2812's, SK6812, SK6805 |
| `WS2812_BYTE_ORDER_RGB`           | WS2812B-2020                  |
```

#define WS2812_TRST_US 80
```

> ### Addressable LED byte order

Reviewer 3: ```suggestion
### Byte Order
```

> The WS2812B-2020 addressable LED has a physically different layout with reversed internal red and green LEDs. 
> Without correction, this causes the two internal LEDs to swap colors and display the wrong color.
> The default setting GRB allows other addressable LEDs to work properly with the correct byte structure (G[7-0]R[7-0]G[7-0]), however the WS2812B-2020 needs the green and red bytes reversed.

Reviewer 3: ```suggestion
Some variants of the WS2812 may have their color components in a different physical or logical order. For example, the WS2812B-2020 has physically swapped red and green LEDs, which causes the wrong color to be displayed, because the default order of the bytes sent over the wire is defined as GRB.

In this case, you can change the byte order with the following define:
```

```c

> #define WS2812_BYTE_ORDER RGB

Reviewer 3: ```suggestion
#define WS2812_BYTE_ORDER WS2812_BYTE_ORDER_RGB
```

#define WS2812_TRST_US 80
```

#### Byte Order

Some variants of the WS2812 may have their color components in a different physical or logical order. For example, the WS2812B-2020 has physically swapped red and green LEDs, which causes the wrong color to be displayed, because the default order of the bytes sent over the wire is defined as GRB.

> In this case, you can change the byte order with the following define:

Reviewer 3: ```suggestion
In this case, you can change the byte order by defining `WS2812_BYTE_ORDER` as one of the following values:
```

| Byte order                        | Known devices                 |
|-----------------------------------|-------------------------------|
| `WS2812_BYTE_ORDER_GRB` (default) | Most WS2812's, SK6812, SK6805 |

> | `WS2812_BYTE_ORDER_RGB`           | WS2812B-2020                  |

Reviewer 4: I think this needs to be updated.
